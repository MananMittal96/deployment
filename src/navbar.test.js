import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import Navbar from './components/navbar';

configure({ adapter: new Adapter() });

describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Navbar />);
  
    expect(component).toMatchSnapshot();
  });
});