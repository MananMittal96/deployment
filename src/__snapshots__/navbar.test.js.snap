// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<nav
  className="nav-wrapper red darken-3"
>
  <div
    className="container"
  >
    <Link
      className="brand-logo"
      to="/"
    >
      Client Management
    </Link>
    <ul
      className="right"
    >
      <li>
        <NavLink
          to="/"
        >
          New Project
        </NavLink>
      </li>
      <li>
        <NavLink
          to="/"
        >
          Log Out
        </NavLink>
      </li>
      <li>
        <NavLink
          className="btn btn-floating pink lighten-1"
          to="/"
        >
          NN
        </NavLink>
      </li>
    </ul>
    <ul
      className="right"
    >
      <li>
        <NavLink
          to="/"
        >
          Signup
        </NavLink>
      </li>
      <li>
        <NavLink
          to="/"
        >
          Login
        </NavLink>
      </li>
    </ul>
  </div>
</nav>
`;
